controllers/project_update.go

```diff
@@ -124,6 +124,16 @@ func UpdateProject(w http.ResponseWriter, r *http.Request) {
 			)
 			return
 		}
+		if tx.Error == gorm.ErrRecordNotFound {
```

Comment:
The execution should not stop here. If the secondary mentor is not found (for example if an empty string is provided), the field should just get unset and no error should be returned.

Reply:
In the [`tProjectUpdateExistent`](https://github.com/kossiitkgp/KWoC-Backend/blob/main/controllers/project_update_test.go#L80) test, 

```expectResponseJSONBodyToBe(t, res, utils.HTTPMessage{StatusCode: http.StatusBadRequest, Message: fmt.Sprintf("Secondary mentor `%s` does not exist.", projUpdateFields.SecondaryMentorUsername)})```

Expects 400 when a mentor is not found.

Reply:
Remove that check.